_data/Pandarati/Pandarati.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Jonathan McKinley
+institution: Florida A&M University 
+profile_pic: mckinley_headshot.jpg
```

Comment:
Incorrect file name, please do one of the following:
1. Change `mckinley_headshot.jpg` to `mckinley_headshot_1.JPG` in the markdown file.
2. Rename `mckinley_headshot_1.JPG` to `mckinley_headshot.jpg`.
I'd suggest you rename the file!